build.xml

```diff
@@ -31,7 +31,7 @@
   </macrodef>
 
   <property name="gwt.apicheck.config"
-            location="tools/api-checker/config/gwt211_212userapi.conf"/>
+            location="tools/api-checker/config/gwt211_213userapi.conf"/>
```

Comment:
I'll make a new api jar instead for 2.12 - or you can? the instructions are in https://github.com/gwtproject/tools/tree/main/api-checker-reference